_layouts/person.html

```diff
@@ -29,6 +32,7 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
     </div>
 
     <div class="person-graduated">
+      {% if person.graduated %}{% unless person.student == true %}<a href="{{ year_link }}">{% endunless %}{% endif %}
```

Comment:
`{% if person.graduated and person.student == false %}`

Reply:
No longer needed as using `{% if person.grad_link %}`